app/services/manifest_validator.rb

```diff
@@ -0,0 +1,28 @@
+class ManifestValidator
+  include JsonSchemaHelper
+
+  attr_reader :content, :errors
```

Comment:
I have removed the `content` accessor in 34ff86f. The `errors` accessor is still valid, since it returns the entire stack of error objects, not just the message strings.